pkg/pb/metainfo.proto

```diff
@@ -15,11 +15,31 @@ import "orders.proto";
 
 // Metainfo it's a satellite RPC service
 service Metainfo {
+    // new API
+    // Bucket
     rpc CreateBucket(BucketCreateRequest) returns (BucketCreateResponse);
     rpc GetBucket(BucketGetRequest) returns (BucketGetResponse);
     rpc DeleteBucket(BucketDeleteRequest) returns (BucketDeleteResponse);
     rpc ListBuckets(BucketListRequest) returns (BucketListResponse);
     rpc SetBucketAttribution(BucketSetAttributionRequest) returns (BucketSetAttributionResponse);
+    // Object
+    rpc BeginObject(ObjectBeginRequest) returns (ObjectBeginResponse);
+    rpc CommitObject(ObjectCommitRequest) returns (ObjectCommitResponse);
+    rpc ListObjects(ObjectListRequest) returns (ObjectListResponse);
+    rpc BeginDeleteObject(ObjectBeginDeleteRequest) returns (ObjectBeginDeleteResponse);
+    rpc FinishDeleteObject(ObjectFinishDeleteRequest) returns (ObjectFinishDeleteResponse);
+    // Segment
+    rpc BeginSegment(SegmentBeginRequest) returns (SegmentBeginResponse);
+    rpc CommitSegment2(SegmentCommitRequest2) returns (SegmentCommitResponse2);
+    rpc MakeInlineSegment(SegmentMakeInlineRequest) returns (SegmentMakeInlineResponse);
+    rpc BeginDeleteSegment(SegmentBeginDeleteRequest) returns (SegmentBeginDeleteResponse);
+    rpc FinishDeleteSegment(SegmentFinishDeleteRequest) returns (SegmentFinishDeleteResponse);
+    rpc ListSegments2(SegmentListRequest) returns (SegmentListResponse);
+    rpc DownloadSegment2(SegmentDownloadRequest2) returns (SegmentDownloadResponse2);
```

Comment:
Can't we have the old API in a different service so that we don't have to name this with a `2` forever? It's unclear to me how that would affect implementing it, though, so maybe that's not possible. Maybe there's some other naming scheme we can use?

Reply:
one way or another it will be in `pb` package so moving this wont work. I was trying to find out if `rpc` names can be changed in a backward-compatible way but spec is saying mainly about fields. We can choose a different naming scheme if it will be needed.

Reply:
Changing name of service methods is not backwards compatible.

Reply:
Yeah, I realized that yesterday

Reply:
I guess we can rename old one. Otherwise we can end up with something like suffixes (e.g. `_v2`) for all new methods. What do you propose for temp name scheme for old methods and messages? `*_v1`? :) I will prepare separate PR for this.

Reply:
I would use `Old` prefix, but a particular choice probably doesn't matter that much.

Reply:
https://github.com/storj/storj/pull/2462